Zip Handler Web App.py

```diff
@@ -0,0 +1,33 @@
+from aiohttp import web
+import zipfile
+
+
+async def handle(request):
+    return web.Response(text='''<!DOCTYPE html>
+<html>
```

Comment:
Instead of making function this bulky, you can create html page with the same text and render it